test/proto/custom_generator/BUILD.bazel

```diff
@@ -100,7 +102,9 @@ toolchain(
 
 scala_proto_toolchain(
     name = "failing_scala_proto_toolchain_def",
-    main_generator = "test.proto.custom_generator.FailingGenerator",
+    named_generators = {
```

Comment:
If only `name_generators` are left then probably we could get rid of `name_` prefix

Reply:
Done